### PR TITLE
New feature

### PR DESCRIPTION
**Pull request by piotrfake**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,5 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+# New feature
```
Comment (piotrowicki) [inline .gitignore]: What feature?

File: .gitignore [hunk 2]
```diff
@@ -10,3 +10,5 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
```
Comment (piotrowicki) [inline .gitignore]: Delete this shit

Comment (piotrowicki) [inline .gitignore]: Or not

File: .gitignore (hunk 1 above)
Comment (piotrowicki) [inline .gitignore]: Some smells bad

Comment (piotrowicki) [inline .gitignore]: Not like this :-1:

Comment (piotrfake) [inline .gitignore]: Why?

Comment (piotrowicki) [inline .gitignore]: Because

Comment (piotrowicki) [inline .gitignore]: smells bad

Comment (piotrowicki) [inline .gitignore]: Comment

Comment (piotrfake) [inline .gitignore]: Good?

Comment (piotrowicki) [inline .gitignore]: :+1:

Comment (piotrowicki) [inline .gitignore]: This is wrong

**Pull request closed without merging**